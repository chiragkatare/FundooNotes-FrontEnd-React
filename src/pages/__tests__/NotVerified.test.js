import React from 'react';
// import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import NotVerified from '../Notverified';
const mockFn = jest.fn();


// configure({ adapter: new Adapter() });

describe('NotVerified', () => {



  it('should render correctly in "debug" mode', () => {
    const component = shallow(<NotVerified debug />);
    // const component = mount(<Login debug />);
    expect(component).toMatchSnapshot();
  });

  it('should render login page Button', () => {
    const component = shallow(<NotVerified debug />);
    const loginButton = component.find('#login');
    expect(loginButton.text()).toEqual('<WithStyles(Button) />');
  });
});