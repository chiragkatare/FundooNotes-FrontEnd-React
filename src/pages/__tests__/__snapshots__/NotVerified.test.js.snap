// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotVerified should render correctly in "debug" mode 1`] = `
<div>
  <WithStyles(Card)
    id="logincard"
  >
    <div
      className="label-forgetpass"
    >
      <div
        className="logo"
      >
        <img
          alt=""
          src={
            Object {
              "ReactComponent": [Function],
              "__esModule": true,
              "default": "logo.svg",
            }
          }
        />
      </div>
    </div>
    <WithStyles(CardContent)>
      <div>
        <WithStyles(Typography)
          align="center"
          component="h4"
          variant="h5"
        >
          !Email Not Verfied!
        </WithStyles(Typography)>
        <WithStyles(Typography)
          align="center"
          component="p"
        >
          Your email is not verified pls Verify your email
        </WithStyles(Typography)>
        <WithStyles(Typography)
          align="center"
          component="p"
        >
          Please Verify your email
        </WithStyles(Typography)>
        <WithStyles(Typography)
          align="center"
          component="p"
        >
          We have sent a link to your email , check you inbox
        </WithStyles(Typography)>
        <WithStyles(Button)
          color="primary"
          fullWidth={true}
          id="login"
          onClick={[Function]}
          type="submit"
          variant="contained"
        >
          Login Page
        </WithStyles(Button)>
      </div>
    </WithStyles(CardContent)>
  </WithStyles(Card)>
</div>
`;
